tox.ini

```diff
@@ -19,13 +20,15 @@ deps =
     dj20: Django>=2,<2.1
     dj21: Django>=2.1,<2.2
     dj22: Django>=2.2,<2.3
+    dj30: Django>=3.0,<3.1
     wt23: wagtail>=2.3,<2.4
     wt24: wagtail>=2.4,<2.5
     wt25: wagtail>=2.5,<2.6
     wt26: wagtail>=2.6,<2.7
     wt27: wagtail>=2.7,<2.8
     wt28: wagtail>=2.8,<2.9
-    wt29: wagtail>=2.9,<2.10
+    wt29: wagtail>=2.9,<3.0
```

Comment:
This line should stay as is, and the following line added instead:
```wt210: wagtail>=2.10,<2.11```

Reply:
amended as requested